### PR TITLE
LPS-65693 Move target platform main into its own module.

### PR DESCRIPTION
**Pull request by shuyangzhou**
Please let the build finish, I have not throughly tested it locally.

### CODE REVIEW

**Comment by shuyangzhou** [comment]
Fixed a typo.

ci:retest

File: modules/core/portal-target-platform-indexer-main/README.markdown [hunk 1]
```diff
@@ -15,6 +15,7 @@ module_framework_base_dir=/home/rotty/AS/liferay-portal/osgi
 jars=(
 	${module_framework_base_dir}/core/com.liferay.portal.bootstrap.jar
 	${module_framework_base_dir}/static/com.liferay.portal.target.platform.indexer.jar
+	${module_framework_base_dir}/../tools/portal-target-platform-indexer-main/com.liferay.portal.target.platform.indexer.main.jar
```
Comment (ZoltanTakacs) [inline modules/core/portal-target-platform-indexer-main/README.markdown]: Hey @shuyangzhou,

Just wondering how will it work on ASes where the dxp_de.war will be deployed so any other than the Tomcat bundle. Will we distribute the tools in an archive or somewhere?

**Comment by shuyangzhou** [comment]
I guess the same question exists for tools/portal-tools-db-upgrade-client.

@brianchandotcom how do you want to solve this?

**Comment by ZoltanTakacs** [comment]
Except the fact that the portal-tools-db-upgrade-client is not crucial for the Patch validation.

**Comment by shuyangzhou** [comment]
Right, I am just saying, we have other tools jars that are not included inside war either. We need a general solution for all of them.

**Comment by ZoltanTakacs** [comment]
Yeah, I agree.

**Pull request closed without merging**